lib/rules/no-deprecated.js

```diff
@@ -64,6 +64,9 @@ module.exports = {
       deprecated.MemberExpression['Perf.printDOM'] = ['15.0.0', 'Perf.printOperations'];
       deprecated.MemberExpression['ReactPerf.getMeasurementsSummaryMap'] = ['15.0.0', 'ReactPerf.getWasted'];
       deprecated.MemberExpression['Perf.getMeasurementsSummaryMap'] = ['15.0.0', 'Perf.getWasted'];
+      // 15.5.0
+      deprecated.MemberExpression['React.createClass'] = ['15.5.0', 'the npm module create-react-class'];
+      deprecated.MemberExpression['React.PropTypes'] = ['15.5.0', 'the npm module prop-types'];
```

Comment:
Pragma would match if there is other thing than `React.*` used right?
I don't think it should match if this is from another object than `React`.